go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/cloudflare/cfssl v0.0.0-20180323000720-5d63dbd981b5
 	github.com/container-storage-interface/spec v1.2.0
 	github.com/docker/distribution v2.7.1+incompatible
-	github.com/docker/docker v20.10.3-0.20220408103430-7ea283fd9166+incompatible // v22.04 dev
+	github.com/docker/docker v20.10.3-0.20221105174132-4f3c5b2568c2+incompatible // v22.04 dev
```

Comment:
We should probably pick a commit (head) from the 22.06 branch in moby/moby, as master is currently used to work on the release after.

Reply:
in any case, the PR in moby/moby I linked is blocking this as both have the same code path. Will update once merged.